tools/install.py

```diff
@@ -159,6 +165,10 @@ def headers(action):
   if sys.platform.startswith('aix'):
     action(['out/Release/node.exp'], 'include/node/')
 
+  if sys.platform.startswith('win'):
```

Comment:
I'm not sure how I did this, but I patched your PR 😵😵😵😵😵😵😵
any way these 3 lines makes the yak almost dateable
(they also made sense, what can you do with the headers without the lib? nothing good I'm quite sure 👩‍⚕️)